Src/VimCore/CoreInterfaces.fs

```diff
@@ -5096,6 +5107,12 @@ and IVimBuffer =
     /// Whether the buffer is readonly
     abstract IsReadOnly: bool with get
 
+    /// Raised Call C# script
+    abstract RaiseCallCSharpScript : CallCSharpScriptEventArgs -> unit
+
+    /// VimBuffer Key Event Intercept
+    abstract KeyIntercept: bool with get,set

```

Comment:
Why do we believe that we need this? The existing API has a lot of hook for consuming key strokes. I don't understand why a new API is needed here? Especially just for running scripts.